### ISSUE TITLE
Replace snapshot tests with Enzyme tests

### ISSUE THREAD

**Issue by turnerhayes** [codeforboston/cliff-effects#911]
This definitely needs discussion, but personally, I have found snapshot tests to be generally useless at best; at my previous job, we started out using them extensively (almost exclusively), but we soon found that they didn't add value for our team; many PRs would contain large snapshot diffs and it was frequently difficult for coders/codecheckers to determine whether the changes were a problem or not. We tended to find ourselves briefly skimming the snapshot diffs or ignoring them. Snapshots also tend to not let you actually determine what exactly they're testing, so it doesn't help you know if your change has broken it.

In my experience, it's much more useful to write Enzyme tests that walk the component hierarchy and make specific assertions about the content.

**Comment by knod** [comment]
Yeah, that lines up with what we've been discovering as well. If all it's doing is checking that the component doesn't crash, you can do a `.not.throw()` or whatever it is. @wacii was key to that discovery process.
